erica/request_processing/erica_input/v2/erica_input.py

```diff
@@ -41,10 +40,18 @@ class ErrorRequestQueue(BaseModel):
     errorMessage: str
 
 
+class Status(Enum):
+    PENDING = "Pending"
```

Comment:
We do not need to show to the outside that a job is pending imo. It should be enough to set it to processing

Reply:
So we don't need the status "Pending"? just processing, failed and completed?